indico/web/client/js/react/components/Datepicker.jsx

```diff
@@ -0,0 +1,78 @@
+// This file is part of Indico.
```

Comment:
Clicking the trigger icon again closes and immediately reopens the date picker. I'd expect it to either not do anything, or close the date picker but not reopen it.